src/main/java/com/stocks/project/model/User.java

```diff
@@ -0,0 +1,34 @@
+package com.stocks.project.model;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.util.Date;
+
+@Entity
+@Table(name = "user")
```

Comment:
The table should not be named user, there will be bugs when working with JPA. Why is this annotation here?